src/components/fadedScrollView/index.tsx

```diff
@@ -11,6 +11,7 @@ import useScrollEnabler from '../../hooks/useScrollEnabler';
 import useScrollReached from '../../hooks/useScrollReached';
 import {forwardRef, ForwardRefInjectedProps} from '../../commons/new';
 import {ScrollView as GestureScrollView} from 'react-native-gesture-handler';
+import {ComponentStatics} from 'src/typings/common';
```

Comment:
Fix import to be relative

Reply:
And while here, let's reorder the imports in this file so packages imports are first and internal imports are second